packages/analytics-browser/generated/amplitude-snippet.js

```diff
@@ -35,6 +41,7 @@
         if (isPromise) return {
           promise: new Promise(getPromiseResult(instance, fn, Array.prototype.slice.call(arguments)))
         };
+        proxyInstance(instance, fn, Array.prototype.slice.call(arguments));
```

Comment:
should it just be `proxy(instance, fn)`?

Reply:
i was trying to look back but couldn't identify what caused the regression (or did it never work?)

Reply:
Thanks, I don't think it ever worked.

I tried using proxy(instance, fn) but it fails to properly handle args.